lambdas/build-user-identity/src/main/java/uk/gov/di/ipv/core/builduseridentity/BuildUserIdentityHandler.java

```diff
@@ -116,17 +116,23 @@ public APIGatewayProxyResponseEvent handleRequest(
 
             var vcs = sessionCredentialsService.getCredentials(ipvSessionId, userId);
 
-            UserIdentity userIdentity =
+            // PYIC-6901 there's probably a better way of getting the target Vot rather than
+            // re-calculating it here. Can we store it in the session in CheckExistingIdentity?
+            // We only actually need this to calculate CI breaches for return codes and the audit
+            // message
+            var targetVot = clientOAuthSessionItem.getLowestStrengthRequestedVot(configService);
```

Comment:
As the comment says here I'm not very happy with this. Is it worth fixing as part of this ticket or leave it for later?

Reply:
I think this method isn't the same since it isn't filtering by gpg45 profile types. I don't think two session-stored attributes would be appropriate: `targetVot` & `targetGpg45Vot`, but think maybe we only need `targetGpg45Vot`?